CONTRIBUTING.md

```diff
@@ -1,11 +1,78 @@
-# Contributing
+# Overview
 
-**Technical Steering Committee (TSC)**
+This project aims to be governed in a transparent, accessible way for the benefit of the community. All participation in this project is open and not bound to corporate affilation. Participants all our bound to the ASWF Code of Conduct.
+
+# Project roles
+
+## Contributor
+
+The contributor role is the starting role for anyone participating in the project and wishing to contribute code.
+
+# Process for becoming a contributor
+
+* Review the [coding standards](http://www.openvdb.org/documentation/doxygen/codingStyle.html) to ensure your contribution is in line with the project's coding and styling guidelines.
+* Sign the Individual CLA and have your organization sign the Corporate CLA.
+* Submit your code as a PR with the appropriate DCO signoff.
+* Have your submission approved by the maintainer(s) and merged into the codebase.
+
+## Committer
+
+The committer role enables the participant to commit code directly to the repository, but also comes with the obligation to be a responsible leader in the community.
+
+### Process for becoming a committer
+
+* Show your experience with the codebase through contributions and engagement on the community channels.
+* Request to become a committer.
+* Have the majority of committers approve you becoming a committer.
+* Your name and email is added to the MAINTAINERS.md file for the project.
+
+### Committer responsibilities
+
+* Monitor email aliases.
+* Monitor Slack (delayed response is perfectly acceptable).
```

Comment:
Perhaps worth removing this line until the decision is made to introduce a Slack channel for OpenVDB to avoid confusion?

Reply:
Good point - do you see interest in having a Slack or IRC-like channel for the group( if there's not one already )?

Reply:
I think it would be good to have something like that, but right now everything is done through email so suggest we just remove this line for now. Ideally, we should mark this down as a future agenda item, perhaps as part of a wider topic about communication. Not high priority though.